### PR TITLE
boards/nucleo-l496zg: doc improvement

### PR DESCRIPTION
**Pull request by krzysztof-cabaj**
### Contribution description

This PR adds to nucleo-l496zg documentation MCU table - similar to those, for example for, Nucleo F103RB, F302R8 or F446RE.

### Testing procedure

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo-l496zg.html
```

### Issues/PRs references

None

### CODE REVIEW

**Review by benpicco — APPROVED** [review]

File: boards/nucleo-l496zg/doc.txt [hunk 1]
```diff
@@ -8,6 +8,28 @@
 The Nucleo-L496ZG is a board from ST's Nucleo family supporting ARM Cortex-M4
 STM32L496ZG ultra-low-pawer microcontroller with 320KiB of RAM and 1 MiB of Flash.
 
+### MCU
+
+| MCU        |   STM32L496ZG      |
+|:---------- |:------------------ |
+| Family     | ARM Cortex-M4      |
```
Comment (benpicco) [inline boards/nucleo-l496zg/doc.txt:15]:
```suggestion
| Family     | ARM Cortex-M4F     |
```

Comment (krzysztof-cabaj) [reply boards/nucleo-l496zg/doc.txt]: Family fixed ... commits squashed.

**Review by maribu — APPROVED** [review]

**Comment by maribu** [comment]
bors merge

Thx for giving our docs some love :)

**Comment by maribu** [comment]
bors merge

**Pull request merged**